### PR TITLE
support cherry-pick automatically with one script

### LINKED ISSUE #11: feat: support cherry-pick automatically or with one script
> /kind process

## Describe the feature
Now, if we want to cherry-pick one PR to old branch, I need to do it manually. 
So we need to do it automatically or one one script

### PR DESCRIPTION
**Pull request by oliviershrek**
Fixes #11 

## Proposed Changes

*
*
*

**Release Note**

```release-note

```

### CODE REVIEW

File: hack/cherry_pick_pull.sh [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+
+STARTINGBRANCH=$(git symbolic-ref --short HEAD)
+declare -r STARTINGBRANCH
+declare -r REBASEMAGIC="${REPO_ROOT}/.git/rebase-apply"
+DRY_RUN=${DRY_RUN:-""}
+REGENERATE_DOCS=${REGENERATE_DOCS:-""}
+UPSTREAM_REMOTE=${UPSTREAM_REMOTE:-upstream}
+FORK_REMOTE=${FORK_REMOTE:-origin}
+MAIN_REPO_ORG=${MAIN_REPO_ORG:-$(git remote get-url "$UPSTREAM_REMOTE" | awk '{gsub(/http[s]:\/\/|git@/,"")}1' | awk -F'[@:./]' 'NR==1{print $3}')}
+MAIN_REPO_NAME=${MAIN_REPO_NAME:-$(git remote get-url "$UPSTREAM_REMOTE" | awk '{gsub(/http[s]:\/\/|git@/,"")}1' | awk -F'[@:./]' 'NR==1{print $4}')}
```
Comment (mrhello369) [inline hack/cherry_pick_pull.sh:40]: This script is from K8s, and do we modified anything?

Comment (oliviershrek) [reply hack/cherry_pick_pull.sh:40]: just copy from k8s

Comment (mrhello369) [reply hack/cherry_pick_pull.sh:40]: Can u give a demo how to use it?

Comment (oliviershrek) [reply hack/cherry_pick_pull.sh:40]: FYI, you can get from https://git.k8s.io/community/contributors/devel/sig-release/cherry-picks.md

**Review by mrhello369 — APPROVED** [review]
/lgtm

**Pull request merged**